### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by ZehraSarigedikDemircioglu**
Can you please pull my solution to master branch?

### CODE REVIEW

File: src/main/java/Solutions/week6/ValidAnagram.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package Solutions.week6;
+
+import java.util.Arrays;
+
+public class ValidAnagram {
```
Comment (IvanRemez) [inline src/main/java/Solutions/week6/ValidAnagram.java:5]: Please add your solutions to the existing Q1_ValidAnagram without creating a new class.

File: src/main/java/Solutions/week6/ValidAnagramStream.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package Solutions.week6;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ValidAnagramStream {
```
Comment (IvanRemez) [inline src/main/java/Solutions/week6/ValidAnagramStream.java:6]: Please add your solutions to the existing Q1_ValidAnagram without creating a new class. Just move your method to that class without deleting anything else. Also the calls from Main method can be transferred over.

**Review by IvanRemez — CHANGES_REQUESTED** [review]

**Comment by ZehraSarigedikDemircioglu** [comment]
I added my solutions to the existed Q1_ValidAnagram and Q2_ValidPalindrome.

**Review by IvanRemez — APPROVED** [review]
Much better! Good job!

**Pull request merged**